homeassistant/helpers/area_registry.py

```diff
@@ -358,8 +360,8 @@ def _handle_floor_registry_update(event: Event) -> None:
 
         self.hass.bus.async_listen(
             event_type=fr.EVENT_FLOOR_REGISTRY_UPDATED,
-            event_filter=_floor_removed_from_registry_filter,
-            listener=_handle_floor_registry_update,
+            event_filter=_floor_removed_from_registry_filter,  # type: ignore[arg-type]
```

Comment:
Why do we need the type ignores, I don't remember?

Reply:
I've already added the basic support for TypeVar defaults to Mypy. That's all that we would need here. Just waiting for a new release now before I can start to update the typing. Should have been out a week ago, so hopefully next week 🤞🏻

I suggest to use `type: ignore`  until then. Will be simple to remove.

https://github.com/python/mypy/issues/16843